### DISCUSSION TITLE
How do I delete multiple messages?

### DISCUSSION THREAD

**Discussion by hashfi-motion** [telegraf/telegraf#1957] [Q&A] [answered]
As the title said. How do I delete multiple message in one API request like this https://core.telegram.org/bots/api#deletemessages? I use Telegraf version `4.15.3`

**Comment by MKRhere** [comment, marked as answer]
Update to `4.16.2`. `ctx.deleteMessages()` and `ctx.telegram.deleteMessages()` are available now.

**Reply by hashfi-motion** [reply]
Thanks for the new release!
